packages/ckeditor5-engine/src/model/node.ts

```diff
@@ -17,9 +17,6 @@ import type Element from './element';
 
 import { CKEditorError, compareArrays, toMap } from '@ckeditor/ckeditor5-utils';
 
-// To check if component is loaded more than once.
-import '@ckeditor/ckeditor5-utils/src/version';
-
```

Comment:
I'm not sure why it's here, and I couldn't find a ticket related to this commit. Version should already be checked by the `utils' package, so do we need this?

I also plan to add a static `version` property to the `Plugin` and `Editor` classes so that they can be compared to each other to specifically detect which plugins are causing the `ckeditor-duplicated-modules` error, so the highlighted check may not be needed.